.gitlab-ci.yml

```diff
@@ -169,6 +169,38 @@ deploy_to_reliability_env:
   rules:
     - when: never # dd-trace-dotnet does not use reliability environment
 
+download-serverless-artifacts:
+  stage: package
+  image: registry.ddbuild.io/docker:20.10.13-gbi-focal
+  tags: [ "arch:amd64" ]
+  needs: []
+  rules:
+    - if: $DOTNET_PACKAGE_VERSION
+      when: on_success
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/' # Manually triggered as artifacts are from the Github release
+      when: manual
+      allow_failure: false
+    - when: delayed # Artifacts come from Azure pipeline, wait a reasonable time before polling
+      start_in: 15 minutes
+  script:
+    - .gitlab/download-serverless-artifacts.sh
+  artifacts:
+    expire_in: 2 weeks
+    paths:
+      - artifacts
+
+aws-lambda-layer:
+  stage: package
+  needs: [download-serverless-artifacts]
+  when: manual
+  trigger:
+    project: DataDog/dd-trace-dotnet-aws-lambda-layer
+    strategy: depend
```

Comment:
Just to be clear, this doesn't actually _publish_ a new layer, right, it just creates it?

Reply:
This creates and publishes a new layer for Sandbox environments, for Production, more steps have to be done on our downstream pipeline, which won't be accidentally triggered from here!